src/main/java/mike/gui/guiHelperMethods.java

```diff
@@ -5,6 +5,8 @@
 import java.io.FileReader;
 import java.io.FileWriter;
 import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;

```

Comment:
Sadly won't let me pinpoint the exact location, but line 27: "import gui.View" needs to be "import mike.gui.View" in order to match the other package names and prevent the build from failing.

Reply:
Or you can just get rid of it, IntelliJ let me know that it's not even used.

Reply:
Yeah I'm just gonna remove them instead, since they're not necessary.